qiskit_experiments/test/mock_iq_helpers.py

```diff
@@ -99,6 +99,34 @@ def iq_phase(self, circuits: List[QuantumCircuit]) -> List[float]:
         """
         return [0.0] * len(circuits)
 
+    def iq_clusters(
+        self,
+        circuits: List[QuantumCircuit],
+        centers: List[Tuple[Tuple[float, float], Tuple[float, float]]],
+        widths: List[float],
+    ) -> List[Tuple[List[Tuple[Tuple[float, float], Tuple[float, float]]], List[float]]]:
```

Comment:
That type hinting of `centers` and the return is very complicated. A bit more description is needed in the docs. I wonder if you can simplify this notation, either by being less precis or defining some data structure but that might be overkill. The code however looks fine.

Reply:
Addressed in dd396e1. A new typing alias `IQPoint=Tuple[float,float]` is used. This makes it shorter, but it isn't *very* short and concise. bae68ce and 5a64569 update the docstrings to explain the return type of `iq_clusters`.